core/drivers/stm32_bsec.c

```diff
@@ -105,10 +111,14 @@
 /* Timeout when polling on status */
 #define BSEC_TIMEOUT_US			1000
 
+#define BSEC_COMPAT		"st,stm32mp15-bsec"
```

Comment:
Minor: the macro just obfuscates the value IMO, I'd rather use the compatible string directly (especially since it is used only once).

Reply:
makes sense. i'll fix.